pipeline/export.py

```diff
@@ -439,8 +439,9 @@ def _export_recording(insert_key, output_dir='./', filename=None, overwrite=Fals
     print('... histology:', end='')
     unit_ccfs = []
     for ccf_tbl in (histology.ElectrodeCCFPosition.ElectrodePosition, histology.ElectrodeCCFPosition.ElectrodePositionError):
-        unit_ccf = (ephys.Unit * ccf_tbl * ccf.CCFAnnotation.proj('annotation') & insert_key
-                    & {'clustering_method': clustering_method}).fetch('unit', 'ccf_x', 'ccf_y', 'ccf_z', 'annotation')
+        unit_ccf = (ephys.Unit * ccf_tbl & insert_key & {'clustering_method': clustering_method}).aggr(
+            ccf.CCFAnnotation, ..., annotation='IFNULL(annotation, "")', keep_all_rows=True).fetch(
```

Comment:
originally, this was the purpose of the synthetic 'CCF Error' region (present in e.g. ecb6a62) - I didn't track what happened to this as we evolved the schema, not sure if we want to revisit - on the one hand this isn't in the 'real' CCF, but on the other hand, it saves needing to do this kind of complex query..

Reply:
I think there are 2 separate concerns here:
1) Whether a unit has a corresponding ccf.CCF entry with it, if not then the `CCF Error` would address this problem
2) Whether a ccf.CCF has an annotation (in ccf.CCFAnnotation) associated with it or not (the problem this PR is addressing, requiring the complex query)
Thoughts/comments welcome

Reply:
as discussed, keeping with current approach for now, can add back 'null region' if needed fairly easily